src/libraries/System.Formats.Tar/tests/TarEntry/TarEntry.Conversion.Tests.Base.cs

```diff
@@ -128,13 +128,13 @@ private TarEntry ConvertAndVerifyEntry(TarEntry originalEntry, TarEntryType entr
                 if (originalEntry.Format is TarEntryFormat.Pax or TarEntryFormat.Gnu)
                 {
                     GetExpectedTimestampsFromOriginalPaxOrGnu(originalEntry, out DateTimeOffset expectedATime, out DateTimeOffset expectedCTime);
-                    CompareDateTimeOffsets(expectedATime, actualAccessTime);
-                    CompareDateTimeOffsets(expectedCTime, actualChangeTime);
+                    Assert.Equal(expectedATime, actualAccessTime);
+                    Assert.Equal(expectedCTime, actualChangeTime);
                 }
                 else if (originalEntry.Format is TarEntryFormat.Ustar or TarEntryFormat.V7)
                 {
-                    CompareDateTimeOffsets(initialNow, actualAccessTime);
-                    CompareDateTimeOffsets(initialNow, actualChangeTime);
+                    AssertExtensions.GreaterThanOrEqualTo(actualAccessTime, initialNow);
```

Comment:
Why is this `GreaterThanOrEqualTo` and not just `Assert.Equal`?

Reply:
In these cases, the `initialNow` timestamp is `DateTimeOffset.UtcNow`, which is generated shortly before invoking the constructors that take the `other` entry. Since the constructors generate the `mtime` timestamp automatically using `UtcNow`, I can't know the _exact expected_ value, but I can at least verify that the value is _larger than_ the timestamp I saved before calling the constructors.

The cases that do an `Equal` comparison are the ones where the _existing_ `mtime` is used as the value to store for `atime` and `ctime`, so I know the exact expected value to compare.